tslint.json

```diff
@@ -27,10 +27,10 @@
       "ignore-bound-class-methods"
     ],
     "switch-default": false,
-    "trailing-comma": [
-      true,
-      { "multiline": "always", "singleline": "never" }
-    ],
-    "no-unused-variable": [true, {"ignore-pattern": "React" }]
+    "no-unused-variable": [true, {"ignore-pattern": "React" }],
+    "trailing-comma": [false],
+    "no-console": [
+      ["error",  ["warn", "error"]]
+    ]
```

Comment:
I’m not sure I understand the implications of this. How does it impact a dev’s workflow, if at all?